CHANGELOG.md

```diff
@@ -17,29 +33,14 @@
   "setup changed" message into all chats we share with the peer #3187
 - do not delete duplicate messages on IMAP immediately to accidentally deleting
   the last copy #3138
-- speed up loading of chat messages #3171 #3194
+- speed up loading of chat messages #3171 #3194 #3173
```

Comment:
```suggestion
- speed up loading of chat messages by a factor of 20 #3171 #3194 #3173
```
Maybe also put it to the top?

Also, we may want to group all the speed-improvements together (also with the ones from "Changes").

Do you know how many people read the changelog? If not that many, that's of course not worth it.

Reply:
yip, we can also move that up, this is really a thing :)